### PR TITLE
Add Ruby 3.1 to CI matrix

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -38,6 +38,7 @@ jobs:
           - ruby: "2.7"
           - ruby: "3.0"
             coverage: "yes"
+          - ruby: "3.1"
```
Comment (ekohl) [inline .github/workflows/test.yml:41]: Should we continue running coverage on the newest Ruby version?
```suggestion
          - ruby: "3.1"
            coverage: "yes"
```

**Comment by bastelfreak** [comment]
Ah I think I tried simplecov on Ruby 3.1 in the past and it didn't work.

**Comment by ekohl** [comment]
Interesting:
```
RuntimeError:
  coverage measurement is already setup
```

In that case I'd be OK with keeping it on 3.0 for now.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -38,6 +38,8 @@ jobs:
...
+
```
Comment (ekohl) [inline .github/workflows/test.yml:42]:
```suggestion
```

**Review by ekohl — APPROVED** [review]
Small nitpick.

File: .github/workflows/test.yml (hunk 2 above)
Comment (bastelfreak) [reply .github/workflows/test.yml]: fixed

**Pull request merged**